test/test_testing.py

```diff
@@ -4333,6 +4333,51 @@ def test_storage_mount(self):
             self.assertEqual(f.read(), 'quux')
 
 
+class TestTestingContainerFilesystemFacade(unittest.TestCase):
```

Comment:
Thanks for adding tests! We should probably also add one that checks that the protocol's method signatures exactly match the `model.Container` methods. Should be able to loop through the protocol methods and use `inspect.signature` and `==` to check? (I haven't tested this.)